src/mango/test/mango.py

```diff
@@ -48,8 +48,8 @@ def __init__(
         dbname,
         host="127.0.0.1",
         port="15984",
-        user="testuser",
-        password="testpass",
+        user="adm",
+        password="pass",
```

Comment:
Why are these changed?

Reply:
Ah, fair enough. But this change should be its own commit then to make it obvious it wasn't an accident.

Reply:
fixed